commands/runner.go

```diff
@@ -54,8 +54,11 @@ func (r *Runner) All() map[string]*Command {
 	return r.commands
 }
 
-func (r *Runner) Use(command *Command) {
+func (r *Runner) Use(command *Command, aliases ...string) {
```

Comment:
I went for the simplest possible solution right now. I don't think we'll use aliases much going forward— these flags-that-are-actually-subcommands is sorta like an edge case. So instead of making true alias support, I'll just stick with the current solution and see whether there will be any need for a more sophisticated system in the future.